public-pool-web/docker-compose.yml

```
@@ -0,0 +1,52 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: public-pool-web_rails-app_1
+      APP_PORT: 3000
+  rails-app:
```

Add newline in between services for better readability.
```suggestion
      APP_PORT: 3000

  rails-app:
```